sagemaker-python-sdk/sparkml_serving_emr_mleap_abalone/sparkml_serving_emr_mleap_abalone.ipynb

```diff
@@ -21,7 +21,7 @@
     "## Setup an EMR cluster and connect a SageMaker notebook to the cluster\n",
```

Comment:
<p>How did you get this list of JARs? It seems brittle, like it could easily change or become outdated. Can the required dependencies be fetched programmatically?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3033/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
The list was identified manually using the `spark-shell`. Changed to use the `spark-shell` config with ivy jar location overwrite